### PR TITLE
Is1165/fix sidecar db conn leak

### LINKED ISSUE #1165: Sidecar sometimes is not able to push files into storage
One of the typical errors that make tutorials fail. I'm not sure whether #1160 fixes this issue too or not.

image

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- Minor modifications ``node_ports`` to avoid connection leak from ``sidecar``
- adds setup to remote debug sidecar service

## Related issue number

Might fix #1165

## How to test

- ``make up-*``
- open osparc and create a pipeline with sleepers
- open admirer -> Process list
- Run sleepers
- Process from application ``sidecar_*`` and ``simcore_sdk..`` should not constantly rise
This is how it looks in my pc
image

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [x] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: services/sidecar/docker/boot.sh [hunk 1]
```diff
@@ -32,15 +32,28 @@ then
...
+then
+  # NOTE: in this case, remote debugging is only available in development mode!
+  # FIXME: workaround since PTVSD does not support prefork subprocess debugging: https://github.com/microsoft/ptvsd/issues/943
+  POOL=solo
+
+elif [[ ${SC_BOOT_MODE} == "debug" ]]
 then
-  # TODO: activate pdb??
   DEBUG_LEVEL=debug
   CONCURRENCY=1
```
Comment (mguidon) [inline services/sidecar/docker/boot.sh]: would be nice to have this as a environment variable. But we are going to refactor the whole thing anyhow. Sometime. If we find time. Maybe.

File: services/web/server/Makefile [hunk 2]
```diff
@@ -16,11 +16,19 @@ install: ## install app in edit mode for development [DEV]
 	@$(VENV_DIR)/bin/pip3 install -r requirements/dev.txt
 
 
-.PHONY: tests
-tests: ## runs all tests [DEV]
+.PHONY: tests tunit tintegration
+tests: tunit tintegration## runs all tests [DEV]
+	# All tests run
+
+tunit:
```
Comment (sanderegg) [inline services/web/server/Makefile]: tests-unit

File: services/web/server/Makefile [hunk 3]
```diff
@@ -16,11 +16,19 @@ install: ## install app in edit mode for development [DEV]
...
 	# running unit tests
 	@$(VENV_DIR)/bin/pytest -vv -x --ff --pdb $(CURDIR)/tests/unit
-	# running integration tests
+
+
+export DOCKER_REGISTRY=local
+export DOCKER_IMAGE_TAG=production
+tintegration:
```
Comment (sanderegg) [inline services/web/server/Makefile]: tests-integration

File: services/sidecar/docker/boot.sh (hunk 1 above)
Comment (pcrespov) [reply services/sidecar/docker/boot.sh]: This is indeed the case with the new template produced with the cookie-cutter

**Pull request merged**